Include/Python.h

```diff
@@ -8,6 +8,8 @@
 #include "pyconfig.h"
 #include "pymacconfig.h"
 
+#include "yk.h"
```

Comment:
Why double quotes include? That'll search the current directory as well as the include dirs.

`#include <yk.h>` would be safer, wouldn't it?

Reply:
Fixed in d3130a3d46.